examples/my_logo_program.txt

```diff
@@ -0,0 +1,9 @@
+fd 200
```

Comment:
Please move this file into another directory like `examples/sample_logo_programs/some_descriptive_name.txt`